app/com/m3/octoparts/aggregator/service/PartResponseLocalContentSupport.scala

```diff
@@ -12,9 +12,14 @@ trait PartResponseLocalContentSupport extends PartRequestServiceBase {
                                  partRequestInfo: PartRequestInfo,
                                  params: Map[ShortPartParam, Seq[String]]): Future[PartResponse] = {
     if (ci.localContentsEnabled) {
-      Future(createPartResponse(ci, partRequestInfo))
+      Future.successful(createPartResponse(ci, partRequestInfo))
     } else {
-      super.processWithConfig(ci, partRequestInfo, params)
+      super.processWithConfig(ci, partRequestInfo, params).map { pr =>
+        if (pr.statusCode.contains(503) && ci.hystrixConfigItem.localContentsAsFallback)
```

Comment:
This is up for debate; not sure if "fallback" should extend to errors coming from legit server responses.